riscv-asm.md

```diff
@@ -525,6 +528,82 @@ Integers in the subinterval [-2048, -1] can also be passed by their (unsigned) a
 in the interval [0xfffff800, 0xffffffff] on RV32I, and
 in [0xfffffffffffff800, 0xffffffffffffffff] on both RV32I and RV64I.
 
+Load Floating Point Immediate
+-------------------
+
+`Zfa` extension has provide `fli.{h|s|d|q}` instruction for loading a specific
+set of floating-point immediates.
+
+The supported floating point immediates are listed in the following table:
```

Comment:
Do we need to duplicate the list here (i.e. can we not refer to the specification)?

Reply:
I would like to hold the this here to let user has a way to quick look up how to write those value in any format (he like), since ISA spec isn't provide this table.

Reply:
reoreded the paragraph, reference to RISC-V ISA spec instead, but put the table in the button of this section and mark as NOTE.